### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

File: pframe/core.py [hunk 1]
```diff
@@ -163,6 +162,7 @@ def __del__(self):
                 pass
 
     def drop(self):
+        import shutil
```
Comment (cowlicks) [inline pframe/core.py]: Why not import this at the module level?

Comment (cpcloud) [inline pframe/core.py]: It's unloaded before `__del__` is called

File: pframe/cframe.py [hunk 2]
```diff
@@ -89,6 +88,7 @@ def __del__(self):
             self.drop()
 
     def drop(self):
+        import shutil
```
Comment (cowlicks) [inline pframe/cframe.py]: Why not module level?

File: dask/bag/core.py [hunk 3]
```diff
@@ -200,17 +203,6 @@ def pluck(self, key, default=no_default):
...
-    @classmethod
-    def from_sequence(cls, *args, **kwargs):
-        raise DeprecationWarning("db.Bag.from_sequence deprecated.\n"
-        "Use db.from_sequence instead.")
-
-    @classmethod
-    def from_filenames(cls, *args, **kwargs):
-        raise DeprecationWarning(
-            "db.Bag.from_filenames deprecated.\n"
-            "Use db.from_filenames instead.")
```
Comment (mrocklin) [inline dask/bag/core.py]: Why delete these?

**Pull request merged**